src/Amido.Stacks.Messaging.Azure.ServiceBus.Tests/UnitTests/Extensions/MessageExtensionsTests.cs

```diff
@@ -1,7 +1,10 @@
+using System;
 using System.Collections;
 using System.Collections.Generic;
 using Amido.Stacks.Messaging.Azure.ServiceBus.Extensions;
 using Amido.Stacks.Messaging.Commands;
+using Amido.Stacks.Messaging.Events;
+using FluentAssertions;
```

Comment:
I am sorry to be a pain, but Shouldly is currently used in this project and therefore should be used for consistency. I prefer FluenttAssertions too but could you please remove this and use the Shouldly syntax.